frontend/app/src/components/defi/Lending.vue

```diff
@@ -7,49 +7,74 @@
+              <dd class="primary--text headline font-weight-bold">
+                <amount-display
+                  :value="totalDepositedUsd"
+                  fiat-currency="USD"
+                  show-currency="symbol"
+                ></amount-display>
+              </dd>
+            </dl>
+          </template>
+          <template #second-col>
+            <dl>
+              <dt class="title font-weight-regular">
+                Effective Savings Rate
+                <v-tooltip bottom>
+                  <template #activator="{ on }">
+                    <v-icon small class="mb-3 ml-1" v-on="on">
+                      fa fa-info-circle
+                    </v-icon>
+                  </template>
+                  <div>
+                    The savings rate across all of the protocols in which<br />
+                    you are actively lending, weighted based on the<br />
+                    relative position in each protocol.
+                  </div>
+                </v-tooltip>
+              </dt>
+              <dd class="primary--text headline font-weight-bold">
+                <amount-display :value="currentDSR"></amount-display>
+                <span style="margin-left: 5px; font-size: 0.8em;">%</span>
+              </dd>
+            </dl>
+          </template>
+          <template #third-col>
+            <dl>
+              <dt
+                class="title font-weight-regular d-flex justify-space-between"
+              >
+                Interest Earned
+                <premium-lock v-if="!premium" class="d-inline"></premium-lock>
+              </dt>
+              <dd
+                v-if="premium"
+                class="primary--text headline font-weight-bold"
+              >
+                <amount-display
+                  :value="totalUsdGainAcrossFilteredAccounts"
+                  show-currency="symbol"
+                  fiat-currency="USD"
+                ></amount-display>
+              </dd>
+            </dl>
+          </template>
+        </stat-card-wide>
       </v-col>
     </v-row>
     <v-row>
       <v-col cols="12">
-        <v-select
-          v-model="selection"
-          class="loans__account-selector"
-          label="Account"
-          return-object
-          :items="dsrBalances"
-          item-text="address"
-        ></v-select>
-      </v-col>
-    </v-row>
-    <v-row v-if="!!selection">
-      <v-col>
-        <stat-card title="DAI locked">
-          {{ selection.balance.dp(decimals) }}
-        </stat-card>
-      </v-col>
-      <v-col>
-        <stat-card title="DAI earned" :locked="!premium">
-          {{ accountGain(selection.address).dp(decimals) }}
-        </stat-card>
-        <stat-card title="approximate $ earned" :locked="!premium">
-          {{ accountGainUsdValue(selection.address).dp(decimals) }}
-        </stat-card>
+        <blockchain-account-selector
+          :addresses="lendingAddresses"
+          :multiple="true"
```

Comment:
@LefterisJP if you want to try out / test the account selection with only 1 account allowed, you can delete this line.
```
          :multiple="true"
```